plugin/metrics/prometheus/metricsstore/reader.go

```diff
@@ -135,9 +132,6 @@ func (m MetricsReader) GetLatencies(ctx context.Context, requestParams *metricss
 
 func buildFullLatencyMetricName(cfg config.Configuration) string {
 	metricName := "latency"
```

Comment:
This can be deleted.

Reply:
The latency histogram metric name was changed to duration as in OTEL migration guide. Can I replace with it?

Reply:
Yup, I expect: `metricName := "duration"`

Reply:
Done!